src/dispatch/signal/models.py

```diff
@@ -48,19 +48,6 @@ class RuleMode(DispatchEnum):
     inactive = "Inactive"
 
 
-assoc_signal_instance_tags = Table(
```

Comment:
This is not used and getting populated in prod, correct?

Reply:
No. It appears that it was dropped in [this migration](https://github.com/Netflix/dispatch/blob/master/src/dispatch/database/revisions/tenant/versions/2023-02-13_b168b50764c7.py#L71) but the code was never removed.

<img width="258" alt="image" src="https://github.com/user-attachments/assets/22cf7658-41d4-4b67-b31e-a93efb581d73">